Uplift/Views/Home/LookingForListCell.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  LookingForListCell.swift
+//  Uplift
+//
+//  Created by Cameron Hamidi on 9/10/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol LookingForListCellDelegate: class {
+    func lookingForCellShouldTagSearch(at tag: Tag, indexPath: IndexPath)
+}
+
+class LookingForListCell: ListCollectionViewCell<Tag, LookingForListItemCell> {
+    
+    // MARK: - Public data vars
+    weak var delegate: LookingForListCellDelegate?
+    var collectionViewWidth: CGFloat = 0
+    var height: CGFloat {
+        return ((collectionViewWidth-48)/2)*0.78
```

Comment:
add spaces between operators and numbers:
`((collectionViewWidth - 48) / 2) * 0.78`

Reply:
This is what was being used previously so I'm not entirely sure, but I think it's just a ratio that design had decided on.